Error: Process completed with exit code 1.

1m 31s
Run npx --no-install next build
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

  ▲ Next.js 14.2.13

   Creating an optimized production build ...
Generated 12 documents in .contentlayer
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/components/analyses/bouwondernemers/BouwondernemersDashboard.tsx:391:17
Type error: Type '{ data: LineSeriesPoint[]; getLabel: (d: LineSeriesPoint) => string; getValue: (d: LineSeriesPoint) => number; getSortValue: (d: LineSeriesPoint) => number; multiLine: boolean; }' is not assignable to type 'IntrinsicAttributes & FilterableChartProps<LineSeriesPoint>'.
  Property 'multiLine' does not exist on type 'IntrinsicAttributes & FilterableChartProps<LineSeriesPoint>'.

  389 |                 }}
  390 |                 getSortValue={(d) => (d as LineSeriesPoint).year}
> 391 |                 multiLine={true}
      |                 ^
  392 |               />
  393 |             </CardContent>
  394 |           </Card>
Error: Process completed with exit code 1.  @claude

--- a/embuild-analyses/src/components/analyses/bouwondernemers/BouwondernemersDashboard.tsx
+++ b/embuild-analyses/src/components/analyses/bouwondernemers/BouwondernemersDashboard.tsx
@@ -388,7 +388,6 @@ function BySectorSection() {
                   return Object.values(rest).reduce((sum: number, v) => sum + v, 0)
                 }}
                 getSortValue={(d) => (d as LineSeriesPoint).year}
-                multiLine={true}
               />
             </CardContent>
           </Card>
@@ -525,7 +524,6 @@ function ByGenderSection() {
                   return Object.values(rest).reduce((sum: number, v) => sum + v, 0)
                 }}
                 getSortValue={(d) => (d as LineSeriesPoint).year}
-                multiLine={true}
               />
             </CardContent>
           </Card>
@@ -651,7 +649,6 @@ function ByRegionSection() {
                   return Object.values(rest).reduce((sum: number, v) => sum + v, 0)
                 }}
                 getSortValue={(d) => (d as LineSeriesPoint).year}
-                multiLine={true}
               />
             </CardContent>
           </Card>
@@ -788,7 +785,6 @@ function ByAgeSection() {
                   return Object.values(rest).reduce((sum: number, v) => sum + v, 0)
                 }}
                 getSortValue={(d) => (d as LineSeriesPoint).year}
-                multiLine={true}
               />
             </CardContent>
           </Card>

